src/operator/concatAll.ts

```diff
@@ -12,6 +11,6 @@ import {MergeAllOperator} from './mergeAll-support';
  *
  * @returns {Observable} an observable of values merged from the incoming observables.
  */
-export function concatAll<T>(): Observable<T> {
+export function concatAll<T>(): T {
```

Comment:
Makes sense, but just bit concerned if someone just read signature might came to think in similar way as I thought originally. Does specifying return type to explicit `Observable<T>` could cause side effect?